src/control/cmd/dmg/auto_test.go

```diff
@@ -566,6 +566,8 @@ engines:
   provider: ofi+verbs
   fabric_iface: ib0
   fabric_iface_port: 31416
+  env_vars:
+  - ABT_THREAD_STACKSIZE=18432
```

Comment:
We probably should not be including this in the output of config gen considering we handle it automatically on reading back the config.

Reply:
I prefer to have ` - ABT_THREAD_STACKSIZE=18432` explicitly listed rather than making some tricks with reverting changes that are common for all configurations.
```
=== RUN   TestAuto_ConfigWrite
    auto_test.go:662: unexpected output config (-want, +got):
          (
          	"""
          	... // 23 identical lines
          	  fabric_iface: ib0
          	  fabric_iface_port: 31416
        + 	  env_vars:
        + 	  - ABT_THREAD_STACKSIZE=18432
          	  pinned_numa_node: 0
          	- targets: 6
          	... // 13 identical lines
          	  fabric_iface: ib1
          	  fabric_iface_port: 32416
        + 	  env_vars:
        + 	  - ABT_THREAD_STACKSIZE=18432
          	  pinned_numa_node: 1
          	disable_vfio: false
          	... // 16 identical lines
          	"""
          )
        
--- FAIL: TestAuto_ConfigWrite (0.01s)
```